pyblish_qml/qml/Perspective/Page.qml

```diff
@@ -8,6 +8,33 @@ Item {
     id: root
 
     property QtObject item
+    property QtObject overview
+    property bool commenting: false
+
+    function setComment(text) {
+        if (overview.state == "") {
+            app.commenting(text, item.name)
+            footer.hasComment = text ? true : false
+        }
+    }
+
+    function getComment() {
+        // GUI may freeze if querying comment while publishing
+        // - note, this triggers commentChanged signal
```

Comment:
Maybe we can do a `if old_comment == new_comment: return` to avoid duplicate signals?

Reply:
Ah, that's just like a note-to-self kind of thing, no duplicated signals from here actually. :)